### DISCUSSION TITLE
Is there a way to change the color of the check inside the checkbox component?

### DISCUSSION THREAD

**Discussion by zataara** [saadeghi/daisyui#1332] [Q&A] [answered]
For the [checkbox](https://daisyui.com/components/checkbox/) component is there a way to customize the color of the check mark inside?

**Comment by saadeghi** [comment, marked as answer]
Yes. Please see https://github.com/saadeghi/daisyui/issues/1119 and https://github.com/saadeghi/daisyui/issues/1286
